app/mailers/authentication_mailer.rb

```diff
@@ -6,4 +6,12 @@ def sign_up_email(to:, token:)
               to: to,
               subject: t('authentication.sign_up.email.subject'))
   end
+
+  def sign_in_email(to:, token:)
+    @magic_link = "#{candidate_interface_welcome_url}/?token=#{token}"
+
+    view_mail(GENERIC_NOTIFY_TEMPLATE,
+              to: to,
+              subject: 'Sign in')
```

Comment:
Can we use a string from the locale here too?

Reply:
there were missing some other locales too, I've added them